demisto_sdk/commands/common/timers.py

```diff
@@ -13,12 +17,41 @@
 
 logger = logging.getLogger('demisto-sdk')
 
-CSV_HEADERS = ['Function', 'Avg', 'Total', 'Call count']
-
 StatInfo = namedtuple("StatInfo", ["total_time", "call_count", "avg_time"])
 
 registered_timers: dict = defaultdict(set)
 
+packs: dict = {}
+
+
+class MeasureType(Enum):
+    FUNCTIONS = 'functions'
+    PACKS = 'packs'
+
+
+MEASURE_TYPE_TO_HEADERS: Dict[MeasureType, Sequence[str]] = {
+    MeasureType.FUNCTIONS: ['Function', 'Avg', 'Total', 'Call count'],
+    MeasureType.PACKS: ['Pack', 'Start Time', 'End Time', 'Total Time']
+}
+
+
+@dataclass()
+class PackStatInfo:
+    start_time: str
+    end_time: Optional[str] = None
+    total_time: Optional[str] = None
+
+    def __iter__(self):
+        return iter(astuple(self))
+
+    def __lt__(self, other):
+        if not isinstance(other, PackStatInfo):
+            raise Exception('Not PackStatInfo')
```

Comment:
Please refrain from raising bare `Exception` objects, a class inheriting it would suit better